StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButtonContent.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  RowButtonContent.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 2/12/25.
+//
+
+import UIKit
+
+/// Defines a view that is contained with a `RowButton`
+protocol RowButtonContent: UIView {
```

Comment:
I'm actually thinking more and more that subclassing would be a good fit here. Could consider this after breaking out the 3 content views.